runtime/rasdump/dmpsup.c

```diff
@@ -515,11 +515,14 @@ configureDumpAgents(J9JavaVM *vm)
 
 	/* Load up the default agents */
 	for (i = 0; i < numDefaultAgents; i++) {
-		char *typeString = defaultAgents[i].type;
-		agentOpts[agentNum].kind = scanDumpType(&typeString);
-		agentOpts[agentNum].flags = J9RAS_DUMP_OPT_ARGS_STATIC;
-		agentOpts[agentNum].args = defaultAgents[i].args;
-		agentNum++;
+		/* disable IBM_JAVADUMP_OUTOFMEMORY, IBM_HEAPDUMP_OUTOFMEMORY, IBM_SNAPDUMP_OUTOFMEMORY for gcpolicy:nogc */
+		if (NULL == strstr(defaultAgents[i].args, "java/lang/OutOfMemoryError")) {
+			char *typeString = defaultAgents[i].type;
```

Comment:
How this would disable core generation for gcpolicy:nogc only?
Looks for me that this code disables core generation for OOM case for all policies. Did I miss something?

Reply:
you are right, I will update it for nogc case only. Thanks